package.json

```diff
@@ -44,16 +44,20 @@
     "devDependencies": {
         "@types/node": "^14.0.14",
         "ajv": "^6.12.2",
+        "npm-run-all": "^4.1.5",
         "ts-node": "^8.10.2",
-        "tsc": "^1.20150623.0",
         "typescript": "^3.9.5",
         "vsce": "^1.77.0",
         "vscode-tmgrammar-test": "0.0.10"
     },
     "scripts": {
         "vscode:prepublish": "test -f ./syntaxes/Scala.tmLanguage.json",
```

Comment:
I don't think this `prepublish` script is actually doing anything useful? I tried messing with the json file but this doesn't result in failures of `vscode:prepublish`

Reply:
@MaximeKjaer , it  just checks if the file `./syntaxes/Scala.tmLanguage.json` exists.